discordbot/stocks/technical_analysis/ta_menu.py

```diff
@@ -0,0 +1,916 @@
+        open: bool
+            whether open price is used. Default: False
+        fast: int
+            fast value. Default: 3
+        slow: int
+            slow value. Default: 10
+        start:
+            date (in date format for start date)
+        end:
+            date (in date format for end date)
+        """
+
+        await adosc_command(ctx, ticker, open, fast, slow, start, end)
+
+    @discord.ext.commands.command(name="stocks.ta.obv")
+    async def obv(self, ctx: discord.ext.commands.Context, ticker="", start="", end=""):
+        """Displays chart with obv of a given stock and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        start:
+            date (in date format for start date)
+        end:
+            date (in date format for end date)
+        """
+
+        await obv_command(ctx, ticker, start, end)
+
+    @discord.ext.commands.command(name="stocks.ta.fib")
+    async def fib(self, ctx: discord.ext.commands.Context, ticker="", start="", end=""):
+        """Displays chart with fib of a given stock and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        start:
+            date (in date format for start date)
+        end:
+            date (in date format for end date)
+        """
+
+        await fib_command(ctx, ticker, start, end)
+
+    @discord.ext.commands.command(name="stocks.ta.view")
+    async def view(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Displays an image of a given stocks data from Finviz and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        """
+
+        await view_command(ctx, ticker)
+
+    @discord.ext.commands.command(name="stocks.ta.summary")
+    async def summary(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Displays a summary of a given stocks ta from FinBrain and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        """
+
+        await summary_command(ctx, ticker)
+
+    @discord.ext.commands.command(name="stocks.ta.recom")
+    async def recom(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Displays a recommendation of a given stocks ta from Tradingview and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        """
+
+        await recom_command(ctx, ticker)
+
+    @discord.ext.commands.command(name="stocks.ta")
+    async def ta(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Stocks Context - Shows Technical Analysis Menu
+
+        Returns
+        -------
+        Sends a message to the discord user with the commands from the stocks.ta context.
+        The user can then select a reaction to trigger a command.
+        """
+
+        if cfg.DEBUG:
+            print(f"!stocks.ta {ticker}")
+
```

Comment:
Doing `!stocks.ta` should generate an error saying that a ticker is required

Reply:
This is checked on the part with if ticker != "":, so this won't be needed to check again.

Reply:
<img width="591" alt="Screenshot 2021-11-06 at 19 08 30" src="https://user-images.githubusercontent.com/25267873/140621070-7ea5e2b4-9247-4172-b4ec-d7ed17c52bb8.png">

The problem is me, as a user, not getting a reply for this

Reply:
I should get an error saying that a ticker is necessary